src/EditorFeatures/TestUtilities/Threading/WpfTestCase.cs

```diff
@@ -19,6 +20,13 @@ public class WpfTestCase : XunitTestCase
     {
         private readonly SemaphoreSlim _wpfTestSerializationGate;
 
+        [EditorBrowsable(EditorBrowsableState.Never)]
+        [Obsolete("Called by the de-serializer; should only be called by deriving classes for de-serialization purposes")]
+        public WpfTestCase()
+        {
+            _wpfTestSerializationGate = new SemaphoreSlim(1);
```

Comment:
:question: @jasonmalinowski This would obviously work for one-off test runs, but do you see any reason why there would be 🐉 if someone tried to run multiple tests from Test Explorer?